include/linux/sched.h

```diff
@@ -34,6 +34,7 @@
 #include <linux/rseq.h>
 #include <linux/seqlock.h>
 #include <linux/kcsan.h>
+#include "linux/double-fetch-detection.h"
```

Comment:
Compared to other LInux headers, this name is a bit too long. Something like "df-detection.h" should be enough.

Again, please only consider these naming conventions when writing patches for the Linux kernel. It is perfectly fine in other projects :)

Reply:
I think they don't like abbreviations too much, because what is "df"? Same for config var.

Maybe at the end of all of it a nicer name will emerge, but I don't have a suggestion right now.

Reply:
Ok, I don't have any strong preference, leaving this up to Merna. Maybe we'll come up with a good name later.